index.html

```diff
@@ -0,0 +1,276 @@
+              <img
+                class="main-program__card__icon"
+                src="assets/icons/talks-icon.png"
+                alt=""
+              />
+              <h3 class="main_program__card__header">Talks</h3>
+              <p class="main-program__card__description">
+                Listen to inspiring speakers from various countries about their
+                experience in African Education.
+              </p>
+            </li>
+
+            <li class="main-program__card">
+              <img
+                class="main-program__card__icon"
+                src="assets/icons/exhibition-icon.png"
+                alt=""
+              />
+              <h3 class="main_program__card__header">Exhibition</h3>
+              <p class="main-program__card__description">
+                Appreciate cutting edge approaches currently being experimented
+                with all thought the African continent.
+              </p>
+            </li>
+
+            <li class="main-program__card">
+              <img
+                class="main-program__card__icon"
+                src="assets/icons/forum-icon.png"
+                alt=""
+              />
+              <h3 class="main_program__card__header">Forum</h3>
+              <p class="main-program__card__description">
+                Have the time to share your thoughts and opinions with experts
+                for each topic.
+              </p>
+            </li>
+
+            <li class="main-program__card">
+              <img
+                class="main-program__card__icon"
+                src="assets/icons/workshop-icon.png"
+                alt=""
+              />
+              <h3 class="main_program__card__header">Workshop</h3>
+              <p class="main-program__card__description">
+                Experience first hand how lessons are delivered to students,
+                rather than simply listening talks about it.
+              </p>
+            </li>
+
+            <li class="main-program__card">
+              <img
+                class="main-program__card__icon"
+                src="assets/icons/ignite-icon.png"
+                alt=""
+              />
+              <h3 class="main_program__card__header">Ignite</h3>
+              <p class="main-program__card__description">
+                Get opportunities to network with affiliates and experts around
+                the continent.
+              </p>
+            </li>
+          </ul>
+          <button class="primary-btn join-btn">
+            Join African Education Summit 2023
+          </button>
+        </div>
+      </section>
+      <section class="speakers">
+        <div class="container">
+          <h2 class="speakers__headline small-underline">Featured Speakers</h2>
+          <div
+            id="speakers__content"
+            class="speakers__content"
+            data-expanded="false"
+          ></div>
+        </div>
+      </section>
```

Comment:
- [ ] The `Featured Speakers` section should have articles organized in two columns of three articles each, kindly make sure to remove the extra speakers and align the speaker's cards properly so it looks like the provided design

|Your website|Provided design|
|-|-|
|![Screenshot_20230529_232805](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/25227022/66d09b2e-acc7-4579-ac1f-5276656d4cbd)|![Screenshot_20230529_233008](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/25227022/8607e9ce-feb9-46c7-b2b8-c1208bfd9202)|